### PR TITLE
Removing specific MariaDB grammar

### PR DESCRIPTION
**Pull request by uurl**
This PR removes specific MariaDB grammar from the MySQL grammar

### CODE REVIEW

File: sql/mysql/Positive-Technologies/MySqlParser.g4 [hunk 1]
```diff
@@ -45,22 +45,22 @@ sqlStatement
     | administrationStatement | utilityStatement
     ;
 
-emptyStatement_
```
Comment (kaby76) [inline sql/mysql/Positive-Technologies/MySqlParser.g4:48]: Please, do not remove my hard work fixing this grammar to work across platforms. The underscore at the end of a symbol is used to allow the grammar to work with Go.

File: sql/mysql/Positive-Technologies/MySqlLexer.g4 [hunk 2]
```diff
@@ -423,7 +426,7 @@ COMMIT:                              'COMMIT';
 COMPACT:                             'COMPACT';
 COMPLETION:                          'COMPLETION';
 COMPRESSED:                          'COMPRESSED';
-COMPRESSION:                         'COMPRESSION';
+COMPRESSION:                         QUOTE_SYMB? 'COMPRESSION' QUOTE_SYMB?;
```
Comment (KvanTTT) [inline sql/mysql/Positive-Technologies/MySqlLexer.g4:429]: I guess the more correct rule is the following:

```suggestion
COMPRESSION:                         'COMPRESSION' | QUOTE_SYMB 'COMPRESSION' QUOTE_SYMB;
```

File: sql/mysql/Positive-Technologies/MySqlParser.g4 [hunk 3]
```diff
@@ -1102,6 +1074,11 @@ unionStatement
       (querySpecificationNointo | queryExpressionNointo)
     ;
 
+lateralStatement
+    : LATERAL (querySpecificationNointo | queryExpressionNointo)
+    | LATERAL '(' (querySpecificationNointo | queryExpressionNointo) ')' (AS? uid)?
```
Comment (KvanTTT) [inline sql/mysql/Positive-Technologies/MySqlParser.g4:1079]:
```suggestion
    : LATERAL ( querySpecificationNointo |
                queryExpressionNointo |
                ('(' (querySpecificationNointo | queryExpressionNointo) ')' (AS? uid)?)
              )
```

**Comment by KvanTTT** [comment]
Unfrotunately, there are errors on CI: https://github.com/antlr/grammars-v4/actions/runs/3635680917/jobs/6141182149 Please take a look and fix.

**Comment by uurl** [comment]
Ready @KvanTTT Thanks 😃

**Comment by teverett** [comment]
@kaby76  @KvanTTT  ready merge?

**Comment by kaby76** [comment]
All set with me.

**Comment by KvanTTT** [comment]
Thank you for the contribution and welcome to grammar repository!

**Pull request merged**